### ISSUE TITLE
Gateway not work

### ISSUE THREAD

**Issue by jiangcuo** [royalapplications/royalapps-community-freerdp#28]
Hi Royal App Team.

The current  wfreerdp version is `3.0.0-dev (1bec63c2f)` and  the gateway options has changed.

```
 /gateway:g:<gateway>[:<port>],u:<user>,d:<domain>,p:<password>,usage-method:
              direct|detect],access-token:<token>,type:[rpc|http[,no-websockets]
              ,extauth-sspi-ntlm]|auto[,no-websockets][,extauth-sspi-ntlm]],
```

The current program still uses the old way, resulting in the current gateway  config not work.
https://github.com/royalapplications/royalapps-community-freerdp/blob/9d984bae1608d90ca2a810eb54d5a4720205ab4e/src/RoyalApps.Community.FreeRdp.WinForms/Configuration/FreeRdpConfiguration.cs#L106

**Comment by StefanKoell** [comment]
Thanks for letting me know. We plan to switch to FreeRDP 3.0 once it has been released. If you can/want to submit a PR in the meantime, you're more than welcome.

cheers,
Stefan

**Comment by StefanKoell** [comment]
Implemented in #29 and #30 
Royal TS will follow soon once past testing

**Comment by StefanKoell** [comment]
Implemented in https://github.com/royalapplications/royalapps-community-freerdp/pull/29 and https://github.com/royalapplications/royalapps-community-freerdp/pull/30
Royal TS will follow soon once past testing
